app/data/team.js

```diff
@@ -88,7 +89,14 @@ class Team {
      * @returns {Number} representing the overall quality of the team
      */
     getTeamOverallQuality(){
-        return 0;
+        let calidadEquipo = 0;
+
+        for (let player of this.listPlayers) {
+            calidadEquipo += player.overall;
+        }
+        calidadEquipo /= getNumberPlayers();
```

Comment:
Falta el this, sabes para qué sirve?
```suggestion
        calidadEquipo /= this.getNumberPlayers();
```

Reply:
Por cierto, teamOverallQuality creo que no es la media de todos los jugadores del equipo **si no el total de la calidad del equipo**. Compruébalo

Reply:
resuelto